prost-build/src/lib.rs

```diff
@@ -1289,6 +1345,7 @@ mod tests {
         let _ = env_logger::try_init();
         Config::new()
             .service_generator(Box::new(ServiceTraitGenerator))
+            .out_dir(std::env::temp_dir())
```

Comment:
Would we want to use something like `tempdir`? It can ensure we don't have test case collisions (if we add more that _would_ collide later) and then cleans up the generated temp directory on drop. Not vital by any means.

Reply:
Yeah, we can fix this in another PR but prob a good idea.